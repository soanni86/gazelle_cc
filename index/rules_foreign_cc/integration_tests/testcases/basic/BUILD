# gazelle:cc_group unit
# gazelle:cc_indexfile generated.ccindex
# gazelle:exclude third-party
