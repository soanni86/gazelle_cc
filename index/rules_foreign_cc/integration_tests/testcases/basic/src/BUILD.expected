load("@rules_cc//cc:defs.bzl", "cc_binary")

cc_binary(
    name = "fmt",
    srcs = ["fmt.cpp"],
    deps = ["//third-party:fmt"],
)
